controllers/vaultdynamicsecret_controller.go

```diff
@@ -293,9 +293,9 @@ func (r *VaultDynamicSecretReconciler) syncSecret(ctx context.Context, c vault.C
 
 	switch method {
 	case http.MethodPut, http.MethodPost:
-		resp, err = c.Write(ctx, path, params)
+		resp, err = c.Write(ctx, vault.NewWriteRequest(path, params, method))
```

Comment:
I'm curious how the `method` param is used? I'm not seeing `Method()` called in the client, for example.

Reply:
Oops, yeah it is only related to the VDS controller. All Write() requests are `PUT`. Fixing.

Reply:
Fixed with a74d5224b88c8929d26ee7380886c267a29d10aa